.circleci/config.yml

```diff
@@ -385,14 +404,14 @@ jobs:
     steps:
       - checkout
       - restore_cache:
-          key: flowmachine-deps-2-{{ checksum "flowmachine/Pipfile.lock" }}
+          key: flowmachine-deps-3-{{ checksum "flowmachine/Pipfile.lock" }}
       - run:
           name: Linting files with black
-          command: PIPENV_PIPFILE=flowmachine/Pipfile pipenv run black --check .
+          command: pip install black && black --check .
 
   lint_js:
     docker:
-      - image: circleci/node
+      - image: cimg/node:lts
```

Comment:
Question: Do we want the lts of Node, or the lastest?

Reply:
lts is fine I think.